From 0c3cedfe0ff319b58db8c384cb48d2bef92bb705 Mon Sep 17 00:00:00 2001
From: Guochun Huang <hero.huang@rock-chips.com>
Date: Mon, 20 Aug 2018 16:07:09 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-tve1030g: prevent some lp4 scale
 frequency

lpddr4 scale frequency when display on, cause the display shake and
vop POST_BUF_EMPTY

Change-Id: Idff6fe039e93a6a938e05524d7dd5ce0d9c72dee
Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
index ffb32e289b83..ebf7a8f23877 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
@@ -220,11 +220,11 @@
 		SYS_STATUS_NORMAL       800000
 		SYS_STATUS_REBOOT       400000
 		SYS_STATUS_SUSPEND      400000
-		SYS_STATUS_VIDEO_1080P  400000
+		SYS_STATUS_VIDEO_1080P  800000
 		SYS_STATUS_VIDEO_4K     800000
 		SYS_STATUS_VIDEO_4K_10B 800000
 		SYS_STATUS_PERFORMANCE  800000
-		SYS_STATUS_BOOST        400000
+		SYS_STATUS_BOOST        800000
 		SYS_STATUS_DUALVIEW     800000
 		SYS_STATUS_ISP          800000
 	>;
-- 
2.35.3

